### ISSUE TITLE
Actuator doesn't use the CORS Configuration with default security configuration and Spring MVC

### ISSUE THREAD

**Issue by mbhave** [spring-projects/spring-boot#11987]
If [Spring Security](https://github.com/spring-projects/spring-security/issues/5011) can add this as a default, we don't need to do anything. If Spring Security doesn't make this a default, we need to see how this can be done. Adding `http.cors()` [here](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/servlet/SpringBootWebSecurityConfiguration.java#L46) wouldn't work for Jersey because there is no `CorsFilter` or `CorsConfigurationSource`

**Comment by michael-simons** [comment]
@wilkinsona pointed me at this ticket. I'm about to write some config for exactly having that feature with Actuator on Jersey. 

Regarding Spring Security: Wouldn't that tangle Actuator and Security again?

**Comment by mbhave** [comment]
If Spring Security added it as a default, Spring Boot wouldn't need to add anything extra and it would just rely on Spring Security's defaults. It wouldn't tangle Actuator and Security because there would be no actuator specific configuration in Spring Boot's security auto-config.

**Comment by mbhave** [comment]
See #9548.

**Comment by mbhave** [comment]
Unblocking this as we have a separate configuration for the management endpoints now in order to expose health and info. So even if Spring Security adds it as a default we would need to call `.cors()` in the management security auto-configuration.

**Comment by mbhave** [comment]
I tagged the wrong issue.

**Comment by mbhave** [comment]
Blocked on https://github.com/spring-projects/spring-framework/issues/26257

**Comment by mbhave** [comment]
I've created a [separate issue for Webflux](#24541).

**Comment by mbhave** [comment]
Reopening this issue because adding `cors` will fail for Jersey. Spring Security throws an exception if `.cors()` is invoked without a `CorsFilter` or a `CorsConfigurationSource` bean. I think we need the change in Spring Security to fix this one.
